app/controllers/concerns/lol_connections.rb

```diff
@@ -1,127 +0,0 @@
-require 'lol'
```

Comment:
Search and user sign up work correctly now.
Friendship functionality still breaks - I know you discontinued efforts to refactor it but are you going to revert it back to the previous working code in this pull request?
Also, the issue I am currently working on (invite to Leaguespec) probably has little to do with the friendship controller since the invitation is for people that have not used LeagueSpec. However, I agree with doing the refactor once though instead of having to do it twice.